MapboxVisionAR.podspec

```diff
@@ -24,8 +24,8 @@ Pod::Spec.new do |s|
 
   s.swift_version = '4.1'
 
-  s.dependency "MapboxVisionNativeAll/AR", "~> 0.3.0"
+  #s.dependency "MapboxVisionNativeAll/AR", "~> 0.3.0"
```

Comment:
Why was the string commented out?

Reply:
That's an accident. thanks for catching that.